tools/ami-cleanup/src/cmd/main_test.go

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright 2024 Gravitational, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package main
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/require"
+	"golang.org/x/mod/semver"
+)
+
+func TestVersion(t *testing.T) {
+	require.True(t, semver.IsValid("v"+Version), "the version string is not a valid semver")
```

Comment:
I'm not sure we need to write tests for other people's code. I assume being under the golang.org umbrella that this project has enough test coverage as-is.

Reply:
> I'm not sure we need to write tests for other people's code

That's not what this is testing. This is testing to ensure that `main.Version` is a valid semver. Given that this is overwritten when building via linker flags I thought it would be a good idea to validate the provided value.

Reply:
Fair point, I'm fine with keeping this here.

We could even rename `TestVersion` to something like `TestTeleportVersionIsSemver`